backend/user/user.py

```diff
@@ -63,10 +63,25 @@ def getOfficeOccupation(self, id_office, month):
                 occupied_days.extend(range(start.day, monthrange(start.year, start.month)[1]+1))
             else:
                 occupied_days.extend(range(start.day, end.day+1))
-        # remove dias duplicados
-        occupied_days = list(dict.fromkeys(occupied_days))
 
         return {
             'success': True,
             'days': occupied_days
         }
+
+    def rent(self, id_office, id_user, rent_days):
+        cursor = self._db.getCursor()
+
+        insert = (
```

Comment:
Inicializa a data de check-in como `None` e tira a data de checkout da tabela já que não tem mais.